samcli/commands/_utils/click_mutex.py

```diff
@@ -0,0 +1,62 @@
+"""
+Module to check mutually exclusive cli parameters
+"""
+from typing import Any, List, Dict, Tuple
+
+import click
+
+
+class Mutex(click.Option):
```

Comment:
Can we name the class to a more specific name?

Reply:
Updated name to ClickMutex which also matches the filename.